### DISCUSSION TITLE
[绳网技术支援]通用加载问题解决方案

### DISCUSSION THREAD

**Discussion by ewpef** [share121/inter-knot#2663] [其他]
非常感谢您使用绳网，绳网技术支援团队致力为用户服务，由于大量用户反映访问问题，为了能最大程度解决您的问题，请按照下面步骤逐步进行排查 *如果以下仍不能解决您的问题，请在评论区指出
第一部分为通用问题解疑，第二部分为特定平台(PC及手机)问题解决方案

第一部分
1.Github访问
由于某些原因，Github在中国访问不稳定，PC用户建议使用Watt Toolkit加速Github或使用加速器
2.脚本问题
截至2024/8/18 [12:00:00时，绳网小助手(绳网跨域助手)的最新版为1.14.1，请检查版本是否为最新，如否请通过脚本管理器(如篡改猴，又称油猴)或[这](https://greasyfork.org/zh-CN/scripts/502874-%E7%BB%B3%E7%BD%91%E8%B7%A8%E5%9F%9F%E5%8A%A9%E6%89%8B)更新

第二部分
PC
浏览器建议使用Chrome，并安装[篡改猴](https://www.tampermonkey.net/)，同时在右上角三个点---扩展程序---管理扩展程序---启用右上角开发者模式
手机
如果您正在使用Via浏览器，Via浏览器的脚本管理器对绳网小助手(绳网跨域助手)的部分功能有问题，导致无法使用，如您执着使用手机访问绳网，请参照以下步骤*可能较为复杂
1.安装[Kiwi Browser](https://kiwibrowser.com/)，这是一款基于[Chromium](https://www.chromium.org/)的手机浏览器，支持类似于Chrome扩展程序
2.根据您的条件选择安装方式(Google Play或Github)，如果选择Github安装，展开第一个版本的Assets选择一个带arm64(或arm)，且不带DEV的版本

![屏幕截图 2024-08-18 130802](https://github.com/user-attachments/assets/dace2fe2-da4f-419a-857d-8093268566a0)

3.安装[篡改猴](https://www.tampermonkey.net/)，请注意：由于Kiwi Browser本身的[问题](https://github.com/Tampermonkey/tampermonkey/issues/2055)，请安装Tampermonkey Legacy (MV2)，而不是另外的版本，否则无法使用 *Kiwi Browser的开发者模式似乎是默认开启的
4.最后正常打开网站安装脚本即可

绳网技术支援团队 HanHan518  请为我的服务做出评价(0-10)
更多问题请在评论区补充

**Comment by shan-dgha** [comment]
好的，大佬<img src="https://upload-bbs.mihoyo.com/upload/2022/11/14/b171cc5b6c8d8b6257a3bf8163fb2680_567796524796359289.png" alt="阿姬得意" width="75" ><img src="https://upload-bbs.mihoyo.com/upload/2022/11/14/b171cc5b6c8d8b6257a3bf8163fb2680_567796524796359289.png" alt="阿姬得意" width="75" >

**Comment by Maple013** [comment]
PC使用Edge浏览器也能正常访问√
